test/pageobjects/mainPage.js

```diff
@@ -1,30 +1,32 @@
-const { config } = require('../../wdio.conf');
 
 class MainPage {
 
-    constructor() {
-        this.url = config.baseUrl;
-        this.consentButtonSelector = '#privacy-policy-banner .banner-button.policy-accept'; 
+    get consentButton() {
+        return $('//div[contains(@class, "policy-accept")]');
     }
 
-    async open() {
-        try {
-            await browser.url(this.url);
-        } catch (error) {
-            console.error('Error opening the page:', error);
-        }
+    get searchResultPlaceholder() {
+        return $('input[placeholder="Search your Address, City or Zip Code"]');
```

Comment:
Try to find better locator

Reply:
Resolved

used locator: "form.search-form > input.search-input"